docs/config_values.rst

```diff
@@ -571,6 +573,49 @@ Options are:
 
 * A string that starts with '$' or '@'.  See `Shorthand Notation`.
 
+quantity_value
+--------------
+
+These represent `astropy.units.Quantity` values, which are a combination of a float and a unit (specifically, an `astropy.units.Unit`).
+
+Options are:
+
+* An `astropy.units.Quantity` value (e.g. '8.7*u.m', where 'u' is the `astropy.units` module)
+* A string interpretable by `astropy.units.Quantity` (e.g. '8.7 m')
+* A dict with:
+
+    * ``type`` = *str* (required)  There is only one valid option:
```

Comment:
There seem to be two valid options.

Reply:
Looks like I mimicked the `sky_value` section, which also technically has two valid options.  I'll fix that one too.